opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationRequestCommon.java

```diff
@@ -57,12 +57,12 @@ public SELF fintech_calls_list_accounts_for_anton_brueckner() {
     // Note that anton.brueckner is typically used for REDIRECT (real REDIRECT that is returned by bank, and not REDIRECT approach in table)
     public SELF fintech_calls_list_accounts_for_anton_brueckner(String bankId) {
         ExtractableResponse<Response> response = withAccountsHeaders(ANTON_BRUECKNER, bankId)
-                    .header(SERVICE_SESSION_ID, UUID.randomUUID().toString())
-                .when()
-                    .get(AIS_ACCOUNTS_ENDPOINT)
-                .then()
-                    .statusCode(HttpStatus.ACCEPTED.value())
-                    .extract();
+            .header(SERVICE_SESSION_ID, UUID.randomUUID().toString())
```

Comment:
Please undo this formatting change - we use 
```
given()
   .something()
.when()
   .another()
.then()
   .expectation()
```
indentation

Reply:
And how do I explain this to codeformatting of intellij?